nilearn/datasets/func.py

```diff
@@ -1669,3 +1669,219 @@ def fetch_cobre(n_subjects=10, data_dir=None, url=None, verbose=1):
 
     return Bunch(func=func, mat_files=mat, phenotypic=csv_array,
                  description=fdescr)
+
+
+def fetch_surf_nki_enhanced(n_subjects=30, data_dir=None,
+                            url=None, resume=True, verbose=1):
+    """Download and load the NKI enhanced resting-state dataset,
+       preprocessed and projected to the surface.
+
+    Parameters
+    ----------
+    n_subjects: int, optional
+        The number of subjects to load from maximum of 102 subjects.
+        By default, 30 subjects will be loaded. If None is given,
+        all 102 subjects will be loaded.
+
+    data_dir: string, optional
+        Path of the data directory. Used to force data storage in a specified
+        location. Default: None
+
+    url: string, optional
+        Override download URL. Used for test only (or if you setup a mirror of
+        the data). Default: None
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        Dictionary-like object, the interest attributes are :
+         - 'func': Paths to functional resting-state images
+         - 'phenotypic': CSV file containing age, dominant hand and sex
+                         for each subject.
+         -
+
+    References
+    ----------
+    :Download: http://fcon_1000.projects.nitrc.org/indi/enhanced/
+
+    Nooner et al, (2012). The NKI-Rockland Sample: A model for accelerating the
+    pace of discovery science in psychiatry. Frontiers in neuroscience 6, 152.
+    URL http://dx.doi.org/10.3389/fnins.2012.00152
+
+    """
+
+    if url is None:
+        url = 'https://www.nitrc.org/frs/download.php/'
+
+    # Preliminary checks and declarations
+    dataset_name = 'nki_enhanced_surface'
+    data_dir = _get_dataset_dir(dataset_name, data_dir=data_dir,
+                                verbose=verbose)
+    ids = ['A00028185', 'A00033747', 'A00035072', 'A00035827', 'A00035840',
```

Comment:
Ok, now its seems downloading well 'A00035072' both 'lh' and 'rh'. Sorry about previous error comment.